easybuild/easyconfigs/g/GLib/GLib-2.54.3-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'GLib'
+version = '2.54.3'
+
+homepage = 'http://www.gtk.org/'
+description = """GLib is one of the base libraries of the GTK+ project"""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['963fdc6685dc3da8e5381dfb9f15ca4b5709b28be84d9d05a9bb8e446abac0a8']
+
+builddependencies = [
+    ('binutils', '2.30'),
+    ('Python', '2.7.15', '-bare'),
+    ('pkg-config', '0.29.2'),
+]
+dependencies = [
+    ('libffi', '3.2.1'),
+    ('gettext', '0.19.8.1', '-libxml2-2.9.8'),
```

Comment:
@furstj Please use the existing `gettext-0.19.8.1-GCCcore-7.3.0.eb` easyconfig file for this, there is no need to add a `gettext` easyconfig that has the `libxml2` version in `versionsuffix` since the `libxml2` is exactly the same as the one used in `gettext-0.19.8.1-GCCcore-7.3.0.eb` .

I understand that you are just updating existing easyconfigs where `gettext-0.19.8.1-GCCcore-6.4.0-libxml2-2.9.7.eb` was used, but that was introduced to break a version conflict on the `libxml2` version in the dependency graph, that is no longer needed now.

So, two things to change:

*) delete `gettext-0.19.8.1-GCCcore-7.3.0-libxml2-2.9.8.eb` from this PR (you can use `:` for this with `--update-pr`, see http://easybuild.readthedocs.io/en/latest/Integration_with_GitHub.html#deleting-easyconfig-files-or-patches)
*) change this line to `('gettext', '0.19.8.1')`